spec/system/task_spec.rb

```diff
@@ -1,27 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
```

Comment:
associationをFactoryに設定すると記載が少なくて済みます。
```suggestion
  let(:task) { create(:task) }
```

Reply:
[`f9c2fbe`](https://github.com/FukiHayashi/rspec_app_exam/pull/1/commits/f9c2fbe65c62ec5d3e87e0b9f6613b624ba415aa)でFactoryにassociationを設定するように修正しました。